src/gbd_mapping/risk_factor.py

```diff
@@ -5331,98 +4431,95 @@
 
-risk_factors.vitamin_a_deficiency.parent = risk_factors.child_and_maternal_malnutrition
+risk_factors.vitamin_a_deficiency.parent = (risk_factors.child_and_maternal_malnutrition)
 
-risk_factors.zinc_deficiency.parent = risk_factors.child_and_maternal_malnutrition
+risk_factors.zinc_deficiency.parent = (risk_factors.child_and_maternal_malnutrition)
 
-risk_factors.tobacco.parent = risk_factors.behavioral_risks
+risk_factors.tobacco.parent = (risk_factors.behavioral_risks)
 risk_factors.tobacco.sub_risk_factors = (risk_factors.smoking, risk_factors.secondhand_smoke,
                                          risk_factors.chewing_tobacco)
 
-risk_factors.smoking.parent = risk_factors.tobacco
+risk_factors.smoking.parent = (risk_factors.tobacco)
 risk_factors.smoking.affected_risk_factors = (risk_factors.low_bone_mineral_density,
                                               risk_factors.high_fasting_plasma_glucose_continuous)
 
-risk_factors.secondhand_smoke.parent = risk_factors.tobacco
+risk_factors.secondhand_smoke.parent = (risk_factors.tobacco)
 
-risk_factors.alcohol_use.parent = risk_factors.behavioral_risks
+risk_factors.alcohol_use.parent = (risk_factors.behavioral_risks)
 
-risk_factors.drug_use.parent = risk_factors.behavioral_risks
+risk_factors.drug_use.parent = (risk_factors.behavioral_risks)
 risk_factors.drug_use.sub_risk_factors = (risk_factors.drug_use_dependence_and_blood_borne_viruses,
                                           risk_factors.suicide_due_to_drug_use_disorders)
 
-risk_factors.metabolic_risks.parent = risk_factors.all_risk_factors
+risk_factors.metabolic_risks.parent = (risk_factors.all_risk_factors)
 risk_factors.metabolic_risks.sub_risk_factors = (risk_factors.high_fasting_plasma_glucose,
                                                  risk_factors.high_systolic_blood_pressure,
                                                  risk_factors.high_body_mass_index,
                                                  risk_factors.low_bone_mineral_density,
                                                  risk_factors.impaired_kidney_function,
                                                  risk_factors.high_ldl_cholesterol)
 
-risk_factors.high_fasting_plasma_glucose.parent = risk_factors.metabolic_risks
+risk_factors.high_fasting_plasma_glucose.parent = (risk_factors.metabolic_risks)
 risk_factors.high_fasting_plasma_glucose.sub_risk_factors = (risk_factors.high_fasting_plasma_glucose_continuous,
                                                              risk_factors.high_fasting_plasma_glucose_categorical)
 
-risk_factors.high_systolic_blood_pressure.parent = risk_factors.metabolic_risks
-risk_factors.high_systolic_blood_pressure.affected_risk_factors = risk_factors.impaired_kidney_function
+risk_factors.high_systolic_blood_pressure.parent = (risk_factors.metabolic_risks)
+risk_factors.high_systolic_blood_pressure.affected_risk_factors = (risk_factors.impaired_kidney_function)
```

Comment:
@collijk this is an example of where text wrap needs to add parentheses. I interpreted what you said in the office to mean just make everything a tuple (that's why I included parent as well). The issue was any bare risk_factor cause it's on repr to be called, which included parent as well. I also changed the repr itself so this is no longer necessary, so we can revert if we want. The parntheses around imports are unneeded but don't break anything

Reply:
I feel like if we put parent in a tuple it suggests that there could be multiple because it makes it look like all the other tuples where there can in fact be multiple. I think it makes more sense just to change the repr like you did and make everything that could possibly contain multiple items (e.g., affected_risk_factors, sub_causes, etc.) a tuple but keep things like parent where there should only ever be 1 not in tuples.

Reply:
backing off parent is fine, but the use of text_wrap for import and for affected_thing is the problem. If we tolerate parens around import it is fixed, otherwise the functionality should be separated

Reply:
It's not fixed though.  
```
>>> a = 'thing'
>>> b = ('thing')
>>> a == b
True
>>> b
'thing'
>>> type(b)
<class 'str'>
```

Reply:
I'll just split them

Reply:
For python to see a tuple with one item you need a trailing comma:
```
>>> a = 'thing'
>>> b = ('thing',)
>>> a == b
False
>>> b
('thing',)
>>> type(b)
<class 'tuple'>
```